ansible/roles/products/hand-e/server/desktop-icons/defaults/main.yml

```
@@ -25,9 +25,11 @@ server_ip: "10.9.11.1"
 right_hand_picture: hand-e.png
 left_hand_picture: hand-e-left.png
 bimanual_hands_picture: hand-e-bimanual.png
-initial_z_value: 0.7551
-arm_x_separation_value: 0.0
-arm_y_separation_value: 0.0
+initial_z: 0.8221
+initial_y_left: 0.0
```

Remember to add the word arm to the initial_y as we discussed:
```suggestion
initial_y_left_arm: 0.0
```